tests/regression/tests/REQUEST-942-APPLICATION-ATTACK-SQLI/942130.yaml

```diff
@@ -107,12 +124,12 @@ tests:
               User-Agent: OWASP ModSecurity Core Rule Set
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             method: GET
-            uri: "/?a=11!=11"
+            uri: "/get?a=This%20is%20like%20no%20other"
             version: HTTP/1.1
           output:
             no_log_contains: id "942130"
-  - test_title: 942130-8
-    desc: "SQL Injection Attack: SQL Tautology"
+  - test_title: 942130-9
+    desc: "SQL Injection Attack: SQL Tautology using MySQL NULL-safe operator <=>"
```

Comment:
Aah! Nice find! I was still wondering about that operator :)

Reply:
:)